app/src/androidTest/java/com/github/se/cyrcle/ui/report/ReportScreenAlertDialogTest.kt

```diff
@@ -26,7 +26,7 @@ class ReportScreenAlertDialogTest {
     composeTestRule
         .onNodeWithTag("AlertDialogContent")
         .assertTextContains(
-            "We take reports very seriously. Please make sure that the information you have mentioned is accurate. If enough users also submit reports on this object, it will be flagged for administrator review. Do you still want to add this Report?")
+            "We take reports very seriously. Please make sure that the information you have mentioned is accurate. Do you want to add this Report?")
```

Comment:
I think here you can use string.xml so that if we change the message we won't have to change this as well.

Reply:
It is in strings.xml! This is just for the text. Don't think it matters for a test?

Reply:
Ohhh yes you are right my bad then !